package.json

```diff
@@ -10,8 +10,8 @@
     "url": "git+https://github.com/daostack/alchemy.git"
   },
   "engines": {
-    "node": "10.16.0",
-    "npm": "6.11.3"
+    "node": "11.15.0",
+    "npm": "6.7.0"
```

Comment:
why npm is downgraded ?

Reply:
Probably because that is the minimum version that will work with node 11.15.0.

Reply:
but before we used 6.11.3 . why not keeping that ?

Reply:
Because this is better for our users, gives them more options.